portal-web/docroot/html/common/themes/password_expiring_soon.jspf

```diff
@@ -14,13 +14,13 @@
  */
 --%>
 
-<c:if test="<%= themeDisplay.isSignedIn() && UserLocalServiceUtil.isPasswordExpiringSoon(user) %>">
-	<aui:script use="liferay-notice">
-		new Liferay.Notice(
+<c:if test='<%= SessionMessages.contains(request, "passwordExpiringSoon") %>'>
```

Comment:
Hey @NolanChan and @dustinryerson,

Did we want to keep the check for ```themeDisplay.isSignedIn()``` still?  I know that ```passwordExpiringSoon``` only gets added to the ```SessionMessages``` in ```LoginPostAction```, so I was not sure if it was worthwhile to keep this check there or not.

Just let me know.  Thanks!